renderdoc/os/os_specific.h

```diff
@@ -437,3 +443,10 @@ inline uint64_t CountLeadingZeroes(uint64_t value);
 #else
 #error Undefined Platform!
 #endif
+
+namespace Android
+{
+bool IsHostADB(const char *hostname);
+uint32_t startAndroidPackageForCapture(const char *package);
+string adbExecCommand(const string &args);
+}
```

Comment:
Yes I was hoping you could point out where it should go. Feel free to move it later.